torchrl/envs/custom/chess.py

```diff
@@ -132,41 +205,126 @@ def _is_done(self, board):
 
     def _reset(self, tensordict=None):
         fen = None
+        pgn = None
         if tensordict is not None:
-            fen = self._get_fen(tensordict).data
-            dest = tensordict.empty()
+            if self.include_fen:
+                fen = self._get_fen(tensordict).data
+                dest = tensordict.empty()
+            if self.include_pgn:
+                fen = self._get_pgn(tensordict).data
```

Comment:
```suggestion
                pgn = self._get_pgn(tensordict).data
```

Also `_get_pgn` doesn't exist, but I figure you realize that